aiohttp/web_urldispatcher.py

```diff
@@ -894,3 +908,85 @@ def freeze(self):
         super().freeze()
         for resource in self._resources:
             resource.freeze()
+
+    def add_routes(self, routes):
+        """Append routes to route table.
+
+        Parameter should be a sequence of RouteDef objects.
+        """
+        # TODO: add_table maybe?
+        for route in routes:
+            route.register(self)
+
+
+def route(method, path, handler, **kwargs):
+    return RouteDef(method, path, handler, kwargs)
+
+
+def head(path, handler, **kwargs):
+    return route(hdrs.METH_HEAD, path, handler, **kwargs)
+
+
+def get(path, handler, *, name=None, allow_head=True, **kwargs):
+    return route(hdrs.METH_GET, path, handler,
+                 allow_head=allow_head, **kwargs)
+
+
+def post(path, handler, **kwargs):
+    return route(hdrs.METH_POST, path, handler, **kwargs)
+
+
+def put(path, handler, **kwargs):
+    return route(hdrs.METH_PUT, path, handler, **kwargs)
+
+
+def patch(path, handler, **kwargs):
+    return route(hdrs.METH_PATCH, path, handler, **kwargs)
+
+
+def delete(path, handler, **kwargs):
+    return route(hdrs.METH_DELETE, path, handler, **kwargs)
+
+
+class RoutesDef(Sequence):
+    """Route definition table"""
+    def __init__(self):
+        self._items = []
+
+    # TODO: add __repr__
```

Comment:
Thank you for suggestion for route definitions should be sorted by insertion order.
It is crucial for router implementation.